org.mwc.debrief.core/src/org/mwc/debrief/core/editors/PlotEditor.java

```diff
@@ -1505,50 +1505,17 @@ else if (input instanceof FileStoreEditorInput)
 
 								final IFile file = ((IFileEditorInput) getEditorInput())
 										.getFile();
-
-								// get the current path (since we're going to be moving the temp
-								// one to it
-								final IPath thePath = file.getLocation();
-
-								// create a backup path
-								final IPath bakPath = file.getFullPath()
-										.addFileExtension("bak");
-
-								// delete any existing backup file
-								final File existingBackupFile = new File(file.getLocation()
-										.addFileExtension("bak").toOSString());
-								if (existingBackupFile.exists())
-								{
-									CorePlugin.logError(Status.INFO,
-											"Existing back file still there, having to delete"
-													+ existingBackupFile.getAbsolutePath(), null);
-									existingBackupFile.delete();
-								}
-
-								// now rename the existing file as the backup
-								file.move(bakPath, true, monitor);
-
-								// move the temp file to be our real working file
-								final File destFile = thePath.toFile().getAbsoluteFile();
-								if (!tmpFile.renameTo(destFile))
-								{
-									FileUtils.moveFile(tmpFile, destFile);
-								}
-
-								// finally, delete the backup file
-								if (existingBackupFile.exists())
-								{
-									CorePlugin.logError(Status.INFO,
-											"Save operation completed successfully, deleting backup file"
-													+ existingBackupFile.getAbsolutePath(), null);
-									existingBackupFile.delete();
+
+								InputStream source = null;
+								try
+								{
+									source = new FileInputStream(tmpFile);
+									file.setContents(source, true, true, monitor);
+								}
+								finally
+								{
+									FileUtil.safeClose(source);

```

Comment:
FileUtil.safeClose has a warning against it:

````
Discouraged access: The method safeClose(Closeable) from the type FileUtil is not accessible due to restriction on required library /home/ian/Apps/eclipse-SDK-3.8.2-linux-gtk-x86_64/plugins/org.eclipse.core.resources_3.8.1.v20121114-124432.jar
````

The class javadoc comment also states that close() should be called & errors handled before calling safeClose().  See the safeClose() javadoc comment here:
http://grepcode.com/file/repo1.maven.org/maven2/com.tqlab.docbook/docbook-highlight/1.0.0/org/eclipse/core/internal/utils/FileUtil.java#254